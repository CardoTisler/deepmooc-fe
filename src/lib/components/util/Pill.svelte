<script lang="ts">
	export let text: string;
	let style = '';
	export { style as class };
</script>

<div class="rounded-full bg-gray-200 px-2 dark:bg-black {style}">
	<span class="text-gray-600 dark:text-yellow-300">{text}</span>
</div>
