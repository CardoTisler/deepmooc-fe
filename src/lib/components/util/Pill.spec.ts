import { render } from '@testing-library/svelte';
import Pill from './Pill.svelte';

describe('Pill.svelte', () => {
	it('should render component', () => {
		const { container } = render(Pill, { text: "test Pill component"});
		expect(container).toMatchSnapshot();
	});
});
 