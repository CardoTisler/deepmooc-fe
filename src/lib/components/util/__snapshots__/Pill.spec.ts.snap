// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Pill.svelte > should render component 1`] = `
<body>
  <div>
    <div
      class="rounded-full bg-gray-200 px-2 dark:bg-black "
    >
      <span
        class="text-gray-600 dark:text-yellow-300"
      >
        test Pill component
      </span>
    </div>
    <!--&lt;Pill&gt;-->
  </div>
</body>
`;
